tests/storage/merge_test.py

```diff
@@ -367,8 +367,8 @@ def make_env(
             sd_type='block',
             format='raw',
             prealloc=sc.SPARSE_VOL,
-            chain_len=2):
-        size = 2 * GiB
+            chain_len=2,
+            size=2 * GiB):
```

Comment:
Can you explain this change?

Reply:
I have removed this change, was not required after changing the `alloc_size` to the capacity in:
```python
    alloc_size = min(chunk_size_mb * MiB, capacity)
```